### PR TITLE
Fixes #29 - added option to set paper size in config file

### LINKED ISSUE #29: Choose/change paper size
The following paper sizes should be supported:

- A3 (this is currently supported but A4 is more important)
- A4 - this is used in Germany
- ?? - this is used in Singapore

This issue requires some research about:
- where do the next conferences of fossasia take place, which paper format do they use
- what are the dimension of the badges used there? name cards, A6, ???

Please share your research here as it is much appreciated to direct the development flow.
You can implement paper-and badge-sizesfor the different use-cases.

### PR DESCRIPTION
**Pull request by djmgit**
Fixes issue #29, added option for paper size. Now users can
specify paper size in config file.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: badges/8BadgesOnA3.svg [hunk 1]
```diff
@@ -10,8 +10,8 @@
    xmlns:xlink="http://www.w3.org/1999/xlink"
    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   width="297mm"
-   height="420mm"
+   width="paper_width"
```
Comment (niccokunzmann) [inline badges/8BadgesOnA3.svg]: Is the file still editable using inkscape? Changing the size will not change the positions of the elements. Could you check?

**Comment by djmgit** [comment]
@jajodiaraghav @niccokunzmann I have added option to set paper size in config file. Paper the given paper size will be mapped to actual dimensions in python script. For now I have added support for A4 and A3. We can make it more flexible by allowing the user to specify custom dimensions in the cconfig file as well. If it sounds good then I would love to iterate on this in subsequent PRs.

File: badges/8BadgesOnA3.svg (hunk 1 above)
Comment (djmgit) [reply badges/8BadgesOnA3.svg]: I checking it, will update PR if necessary  ASAP

**Comment by niccokunzmann** [comment]
@djmgit okay, so I will merge it.

**Pull request merged**